### PR TITLE
Unbreak build on FreeBSD after qtermwidgets update

### PR DESCRIPTION
**Pull request by rhurlin**
## Description

After qtermwidgets has been updated in the GRASS provider, the build on FreeBSD is broken with the following error:

.../src/plugins/grass/qtermwidgets/kpty.cpp:98:11:fatal error:'utmp.h' file not found.

The patch circumstances this 'classic' utempter problem for FreeBSD.

### CODE REVIEW

**Review by nyalldawson — APPROVED** [review]

**Comment by rhurlin** [comment]
> Can you submit this upstream too please?

Thanks for this very fast approval :)

I am afraid I have not write access to this repository. Loïc Bartoletti offered to do this for me.

**Comment by nyalldawson** [comment]
By upstream I mean a pr at https://github.com/lxqt/qtermwidget

I'll merge this one after the ci finishes its run.

**Comment by rhurlin** [comment]
> By upstream I mean a pr at https://github.com/lxqt/qtermwidget

Ah ok, I had misunderstood that. 

In the meantime, I contacted our FreeBSD maintainer for the KDE ports about the qtermwidgets patch. He's a good judge of whether this FreeBSD-specific patch is otherwise relevant enough to be committed upstream.

> 
> I'll merge this one after the ci finishes its run.

Thanks for the merge :)

**Comment by rhurlin** [comment]
> Can you submit this upstream too please?

I just opened pull request [qtermwidget #503](https://github.com/lxqt/qtermwidget/pull/503) :)

**Pull request merged**
